sonic_api.yaml

```diff
@@ -215,26 +222,31 @@ paths:
 #----------------------------------------------
 # Operations
 #----------------------------------------------
+# schema_violation | Schema violation: must have required property 'schema' (paths > /operations/ping > post > parameters > 0)
+# schema_violation | Schema violation: must NOT have additional properties (paths > /operations/ping > post > parameters > 1)
   '/operations/ping':
     post:
+      operationId: Ping
       summary: pings an ip address from a VRF context, if provided
       description: Returns a JSON response object with packets transmitted, received, maxRTT, minRTT, avgRTT. When specified vnet_id doesn't exist in the config_db, returns 404 Object not found.
       parameters:
-        - name: ip_address
+        - name: pingRequest
```

Comment:
so openapi doesn't appear to allow multiple 'body' parameters, as there is no way to put multiple body elements in rest. I tried to move this to being an object with the body params as elements. This part is one of the problematic apis that I don't know the proper fix for.

Reply:
This will result in the following body:

{"ip_address": string_here, "vnet_id": string_here, "count": string_here}

Which I think is what is required?

Reply:
I looked up name for the body portion and found this on openapi's site: 

> "Note: The name of the body parameter is ignored. It is used for documentation purposes only. For a more modular 
> style, you can move the schema definitions to the global definitions section and refer to them by using $ref:" 

https://swagger.io/docs/specification/2-0/describing-request-body/ So the name isn't used if the in: is body